src/bun.js/api/server.zig

```diff
@@ -5332,6 +5344,10 @@ pub fn NewServer(comptime NamespaceType: type, comptime ssl_enabled_: bool, comp
 
         extern fn JSSocketAddress__create(global: *JSC.JSGlobalObject, ip: JSValue, port: i32, is_ipv6: bool) JSValue;
 
+        pub fn socketId(_: *ThisServer, request: *JSC.WebCore.Request) JSC.JSValue {
```

Comment:
This doesn't need to be a public, user-facing method.

This can instead be a private function only used in node:http.